.rubocop.yml

```diff
@@ -2,4 +2,7 @@ Documentation:
   Enabled: false
 
 LineLength:
-  Max: 99
+  Max: 150
```

Comment:
Don't change de max line length of the whole rubocop, you can change it on the spec files only like:
```
Metrics/LineLength:
  Exclude:
    - spec/**/*
```